io-engine/src/grpc/v1/pool.rs

```diff
@@ -266,10 +298,16 @@ impl PoolRpc for PoolService {
         &self,
         request: Request<ExportPoolRequest>,
     ) -> GrpcResult<()> {
-        self.locked(
-            GrpcClientContext::new(&request, function_name!()),
+        let ctx = GrpcClientContext::new(&request, function_name!());
+        let args = request.into_inner();
+
+        self.serialize_grpc(
+            ctx,
+            Some(args.clone().name),
+            self.init_subsystem().clone(),
+            true,
```

Comment:
export shouldn't be global operation. @tiagolobocastro ?

Reply:
hmm not sure.
Which operations do we want to be global?